t3-2015/week_six.md

```diff
@@ -0,0 +1,33 @@
+Week 6 News letter
+
+This week here at Davinci Coders, it was much of the same. Yet, at the same time, all very new… We got back our grades 
+for the first assessment, which was over making classes and methods, specifically a Credit Card class with payment methods. 
```

Comment:
Can you pull out the Credit Card stuff?  Maybe `specifically creating a class with a couple methods`?